crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -29,6 +29,7 @@ const AUTHENTICITY_NOT_GUARANTEED: &str =
 const UNVERIFIED_IDENTITY: &str = "Encrypted by an unverified user.";
 const UNSIGNED_DEVICE: &str = "Encrypted by a device not verified by its owner.";
 const UNKNOWN_DEVICE: &str = "Encrypted by an unknown or deleted device.";
+pub const SENT_IN_CLEAR: &str = "Sent in clear.";
```

Comment:
Not a fan of these and I know you aren't either. Hope we'll find a bit of time to migrate away from them soon.